components/insights/insights-hero.tsx

```diff
@@ -0,0 +1,20 @@
+const InsightsHero = () => {
+  return (
+    <section className='bg-gradient-to-r from-blue-900 to-indigo-800 py-16 md:py-24'>
+      <div className='container mx-auto px-4'>
+        <div className='mx-auto max-w-3xl text-center'>
+          <h1 className='mb-4 text-4xl font-bold leading-tight tracking-tight text-white md:text-5xl lg:text-6xl'>
+            Insights & Expertise
+          </h1>
+          <p className='text-lg font-light leading-relaxed text-blue-100 md:text-xl'>
+            Dive into our thought leadership, industry analysis, and technical
+            insights. Explore the latest trends and best practices in cloud,
+            design, data, and security.
```

Comment:
please update the last sentence to:
`Explore the latest trends and best practices in cloud, design, data, and security across various sectors like energy, e-commerce, finance, agriculture e.t.c`